news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md

```diff
@@ -0,0 +1,127 @@
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/338/Songs/Sobrem__NIWASHI_-_A_fight_in_Tunisia.mp3">
+    </audio>
+</div>
+
+## Round of 32 results
+
+You can find the full results right here. Yup, go on, take a look.
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| **Philippines** ::{ flag=PH }:: | **5** | 0 | ::{ flag=CZ }:: Czechia |
+| **Vietnam** ::{ flag=VN }:: | **5** | 1 | ::{ flag=UA }:: Ukraine |
+| **Hong Kong** ::{ flag=HK }:: | **5** | 4 | ::{ flag=ID }:: Indonesia |
+| **Singapore** ::{ flag=SG }:: | **5** | 3 | ::{ flag=PL }:: Poland |
+| **China** ::{ flag=CN }:: | **5** | 0 | ::{ flag=FI }:: Finland |
+| **Japan** ::{ flag=JP }:: | **5** | 1 | ::{ flag=FR }:: France |
+| **South Korea** ::{ flag=KR }:: | **5** | 0 | ::{ flag=PE }:: Peru |
+| **United States** ::{ flag=US }:: | **5** | 0 | ::{ flag=NL }:: Netherlands |
+| **United Kingdom** ::{ flag=GB }:: | **5** | 1 | ::{ flag=AR }:: Argentina |
+| **Canada** ::{ flag=CA }:: | **5** | 2 | ::{ flag=AU }:: Australia |
+| **Chile** ::{ flag=CL }:: | **5** | 0 | ::{ flag=NZ }:: New Zealand |
+| **Taiwan** ::{ flag=TW }:: | **5** | 2 | ::{ flag=MX }:: Mexico |
+| **Thailand** ::{ flag=TH }:: | **5** | 0 | ::{ flag=RO }:: Romania |
+| **Malaysia** ::{ flag=MY }:: | **5** | 1 | ::{ flag=SE }:: Sweden |
+| **Russian Federation** ::{ flag=RU }:: | **5** | 1 | ::{ flag=ES }:: Spain |
+| **Brazil** ::{ flag=BR }:: | **5** | 0 | ::{ flag=IT }:: Italy |
+
+## Match highlights
+
+### ::{ flag=HK }:: Hong Kong vs. ::{ flag=ID }:: Indonesia
+
+[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115212102)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2233665337&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Indonesia's renewed roster with 6 new faces had an interesting first match against the higher-seeded Hong Kong. If you don't feel like watching the whole match, these highlights should contain the gist:
+
+- [SHRINE 418 - Little Little Princess](https://osu.ppy.sh/beatmapsets/2234579#mania/4746363) (RC3) @ [15:40](https://www.twitch.tv/videos/2233665337?t=00h15m40s) - The first breakpoint of the match, a cohesive three-way FC from Indonesia to take the match lead.
+- [ShinRa-Bansho - Brain Rigid Girl](https://osu.ppy.sh/beatmapsets/2234595#mania/4746400) (LN2) @ [41:05](https://www.twitch.tv/videos/2233665337?t=00h41m05s) - A clutch save by Hong Kong on match point, and the closest pick in this entire RO32!
+- [SiLiS - Hebe](https://osu.ppy.sh/beatmapsets/2234586#mania/4746375) (HB2) @ [52:18](https://www.twitch.tv/videos/2233665337?t=00h52m18s) - The tiebreaker, the essence of all tournaments, the final consistency test for Hong Kong to secure their victory.
```

Comment:
Supposed to be

- [lapix - Flying Castle (Extended Mix)](https://osu.ppy.sh/beatmapsets/2234596#mania/4746402) (TB)

Reply:
9372f371f81260997e9bfb068f64826e3d23c7d8